### ISSUE TITLE
Extended event table search

### ISSUE THREAD

**Issue by MikhailSmirnou** [zalando-incubator/nakadi-ui#28]
Currently the table https://nakadi-ui-staging.aruha-test.zalan.do/#types?filter=reservation&page=2&reverse=True&sortBy=owning_application may only be sorted by, for example, category or owning application. Would be convenient to list only events belonging to some particular category and/or owning application.

**Comment by SergKam** [comment]
If I understand it correctly you want a column level filtering. 
Maybe like this 
![jquery-plugin-for-multi-column-table-sorting-filtering-dynamitable](https://user-images.githubusercontent.com/3240529/47214007-f2671180-d39c-11e8-860e-970d81882bcb.jpg)

Or it can be some of the filtering/searching text formats like `$category=data &&  $owning_application=myapp` less user-friendly but more powerful.

**Comment by MikhailSmirnou** [comment]
Yes, something like this. For me the "Owning Application" and "Category" fields would be enough. May be other fields, like "Updated" (with time range) would make sense to filter for.

> `$category=data && $owning_application=myapp`

You mean, the user should be adding such conditions to URL manually? 
I'm afraid, very few people will know about that. Only if the proper hint will be added and will be noticeable on a page.

**Comment by SergKam** [comment]
Sorry, `$category=data && $owning_application=myapp` not in URL but in filter input field, like extended filter in Jira or Saclyr.
